Makefile.container

```
@@ -0,0 +1,14 @@
+include $(WORKSPACE_TOP)/lb_containers/Makefile.container
```

missing copyright and license notice

---

should error out if WORKSPACE_TOP is not defined